src/notifications/information-message.js

```diff
@@ -0,0 +1,8 @@
+import dedent from 'dedent';
+
+
+export const SCREEN_RECORDING_PERMISSION_REQUEST = dedent `
```

Comment:
Version approved by Dirk is:

TestCafe requires permission to record the screen. Open 'System Preferences > Security & Privacy > Privacy > Screen Recording' and check 'TestCafe' in the application list.

Press any key to retry.

Reply:
Yeah, right, my mistake

Reply:
Yeah, right, my mistake 😢